build-tools/scripts/BuildEverything.mk

```diff
@@ -109,4 +109,6 @@ package-oss $(ZIP_OUTPUT):
 	if [ -d bin/Release/bin ] ; then cp tools/scripts/xabuild bin/Release/bin ; fi
 	if [ ! -d $(ZIP_OUTPUT_BASENAME) ] ; then mkdir $(ZIP_OUTPUT_BASENAME) ; fi
 	if [ ! -L $(ZIP_OUTPUT_BASENAME)/bin ] ; then ln -s ../bin $(ZIP_OUTPUT_BASENAME) ; fi
-	zip -r "$(ZIP_OUTPUT)" $(ZIP_OUTPUT_BASENAME) $(_BUNDLE_ZIPS:%=--exclude %)
+	cp -a build-tools/debian-metadata $(ZIP_OUTPUT_BASENAME)/debian
+	cd $(ZIP_OUTPUT_BASENAME) && DEBEMAIL="Xamarin Public Jenkins (auto-signing) <releng@xamarin.com>" dch --create -v $(PRODUCT_VERSION).$(-num-commits-since-version-change) --package oss-xamarin.android --force-distribution --distribution alpha "New release - please see git log for $(GIT_COMMIT)"
```

Comment:
This step is also executed on macOS, and this command would break macOS builds (because macOS doesn't have the `dch` command).

This should *probably* follow the spirit of the [`make prepare`](https://github.com/xamarin/xamarin-android/blob/d91ef1a/Makefile#L81) target: make it a multi-target, and have OS-specific "overloads" of the target:

    package-oss $(ZIP_OUTPUT)::
    	if [ -d bin/Debug/bin ]   ; then cp tools/scripts/xabuild bin/Debug/bin   ; fi
      	if [ -d bin/Release/bin ] ; then cp tools/scripts/xabuild bin/Release/bin ; fi
      	if [ ! -d $(ZIP_OUTPUT_BASENAME) ] ; then mkdir $(ZIP_OUTPUT_BASENAME) ; fi
      	if [ ! -L $(ZIP_OUTPUT_BASENAME)/bin ] ; then ln -s ../bin $(ZIP_OUTPUT_BASENAME) ; fi
    	zip -r "$(ZIP_OUTPUT)" $(ZIP_OUTPUT_BASENAME) $(_BUNDLE_ZIPS:%=--exclude %)

    ifeq ($(OS),Linux)
    $(ZIP_OUTPUT)::
    	cp -a build-tools/debian-metadata $(ZIP_OUTPUT_BASENAME)/debian
    	cd $(ZIP_OUTPUT_BASENAME) && DEBEMAIL="Xamarin Public Jenkins (auto-signing) <releng@xamarin.com>" dch --create -v $(PRODUCT_VERSION).$(-num-commits-since-version-change) --package oss-xamarin.android --force-distribution --distribution alpha "New release - please see git log for $(GIT_COMMIT)"
    endif # Linux

This would prevent the macOS build from failing, while allowing Linux to also build Debian packages. (It would thus promptly fail when someone tried to build on Centos...)